src/test/java/com/googlecode/lanterna/gui2/ComboBoxTest.java

```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) throws IOException, InterruptedException
 
     @Override
     public void init(WindowBasedTextGUI textGUI) {
-        final BasicWindow window = new BasicWindow("ComboBoxTest");
+        final BasicWindow window = new BasicWindow();
```

Comment:
Woops! Those shouldn't have gone in! My bad.

I'll bring up a PR now which reverts their changes